service.subtitles.napisy24pl/changelog.txt

```diff
@@ -0,0 +1,48 @@
+3.0.0 - by CaTz (10/01/2020)
```

Comment:
Changelog is no longer exposed on the kodi interface in favor of the `<news>` tag. Please add it if you wish (only mentioning the last version) and remove the changelog from the submission